modules/overviews/spec/features/managing_overview_page_spec.rb

```diff
@@ -0,0 +1,216 @@
+# frozen_string_literal: true
+
+#-- copyright
+# OpenProject is an open source project management software.
+# Copyright (C) the OpenProject GmbH
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+require "spec_helper"
+
+require_relative "../support/pages/overview"
+
+RSpec.describe "Overview page managing", :js, with_flag: { new_project_overview: false } do
+  let!(:type) { create(:type) }
+  let!(:project) { create(:project, types: [type], description: "My **custom** description") }
+  let!(:open_status) { create(:default_status) }
+  let!(:created_work_package) do
+    create(:work_package,
+           project:,
+           type:,
+           author: user)
+  end
+  let!(:assigned_work_package) do
+    create(:work_package,
+           project:,
+           type:,
+           assigned_to: user)
+  end
+
+  let(:permissions) do
+    %i[manage_dashboards
```

Comment:
N.B. we're using the new permission, `manage_dashboards` here, not ``manage_overview`